pykokkos/core/runtime.py

```diff
@@ -3,14 +3,18 @@
 import sys
 from typing import Any, Callable, Dict, Optional, Tuple, Type, Union, List
 import sysconfig
-
+import ast
 import numpy as np
```

Comment:
Imports should be done in alphabetical order, so move `import ast` to the top. Also keep the newline between the standard library imports and the numpy import